<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="9285" started-at="2021-10-14T20:09:51 CEST" finished-at="2021-10-14T20:10:00 CEST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="9285" started-at="2021-10-14T20:09:51 CEST" finished-at="2021-10-14T20:10:00 CEST">
      <class name="tests.MealItemTest">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:tests.MealItemTest@76f2b07d]" name="beforeMethod" is-config="true" duration-ms="3996" started-at="2021-10-14T20:09:51 CEST" finished-at="2021-10-14T20:09:55 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="clearCartTest()[pri:0, instance:tests.MealItemTest@76f2b07d]" name="clearCartTest" duration-ms="5222" started-at="2021-10-14T20:09:55 CEST" finished-at="2021-10-14T20:10:00 CEST">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[data\Data.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: data\Data.xlsx (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at tests.MealItemTest.clearCartTest(MealItemTest.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearCartTest -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:tests.MealItemTest@76f2b07d]" name="afterMethod" is-config="true" duration-ms="1" started-at="2021-10-14T20:10:00 CEST" finished-at="2021-10-14T20:10:00 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- tests.MealItemTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
